.gitignore

```diff
@@ -2,3 +2,4 @@
 firebase-debug.log
 .idea
 node_modules
+backup.html
```

Comment:
I assume you added this because you made a backup on a file while editing. 
Can you kindly remove this line?

You can perhaps move the file outside the folder, if you wish to avoid committing it.

Reply:
done